### PR TITLE
feat/TEC-21: Improved optional PR title parameter.

### PR DESCRIPTION
**Pull request by gergo-dryrun**
Work started on feat/TEC-21

### CODE REVIEW

File: aliases/start-work/README.md [hunk 1]
```diff
@@ -36,8 +41,11 @@ gh start-work <JIRA_TICKET> [ISSUE_TYPE]
 # Start work on a feature
 gh start-work PROJ-123
 
-# Start work on a bug fix
-gh start-work PROJ-456 fix
+# Start work on a feature with just a PR title (uses default feat type)
+gh start-work PROJ-123 "Add login functionality"
```
Comment (nickdecooman) [inline aliases/start-work/README.md:45]: Unrelated to this PR, but could we potentially shorten `start-work` to eg. `start` ?

File: aliases/start-work/alias.sh [hunk 2]
```diff
@@ -40,7 +41,7 @@ gh alias set --clobber start-work '!f() { \
   REPO=$(git config --get remote.origin.url | sed "s/.*github.com[:/]\(.*\)\.git/\1/"); \
   BRANCH_NAME="${ISSUE_TYPE}/${JIRA_TICKET// /_}"; \
   git checkout -b "$BRANCH_NAME" && \
-  echo -e "\n# ${BRANCH_NAME}" >> .changes.md && \
+  echo "\n# ${BRANCH_NAME}" >> .changes.md && \
```
[the remark this answers is not in the document]
Comment (nickdecooman) [reply aliases/start-work/alias.sh:44]: Ah... is it to populate the branch with a commit so you can create a draft PR?

[the remark this answers is not in the document]
Comment (gergo-dryrun) [reply aliases/start-work/alias.sh:44]: Yep. Needed something for it.

File: aliases/start-work/README.md (hunk 1 above)
Comment (gergo-dryrun) [reply aliases/start-work/README.md:45]: Will create a separate PR.

**Pull request merged**